components/form/LabeledSelect.vue

```diff
@@ -55,6 +55,10 @@ export default {
       default: null,
       type:    String
     },
+    pushTags: {
```

Comment:
I couldn't see this used in the PodAffinity LabeledSelect. Was it to fix `./chart/monitoring/StorageClassSelector.vue`?

Reply:
This accidentally got in. It doesn't do any harm, but since it's not necessary for this PR, I'll remove it.